libkbfs/folder_branch_ops.go

```diff
@@ -1914,6 +1915,19 @@ func (fbo *folderBranchOps) finalizeMDWriteLocked(ctx context.Context,
 
 	var mdID MdID
 
+	if err = EnableDelayedCancellationWithGracePeriod(
```

Comment:
Yeah I think it's because Git doesn't expect EINTR from some of I/O calls. Taru mentioned some manual pages where those calls don't have EINTR listed as possible errors, so Git's behavior is expected in this case.

I'll add comments!